src/FileCopyPlugin.php

```diff
@@ -66,6 +66,31 @@ public function onPostCmd(Event $event): void
         if (file_exists("$vendorDir/mklkj/tinymce-i18n/langs6")) {
             $this->copyDirectory("$vendorDir/mklkj/tinymce-i18n/langs6", "$projectDir/public/js/tinymce/langs", $io);
         }
+
+        // Get all maho modules
+        $localRepo = $composer->getRepositoryManager()->getLocalRepository();
+        $packages = $localRepo->getPackages();
+        $mahoModules = array_filter($packages, function($package) {
+            return in_array($package->getType(), ['magento-module', 'maho-module'], true);
+        });
+
+        // Copy public folders from module packages
+        foreach ($mahoModules as $package) {
+            $packageName = $package->getName();
+            $packagePath = "$vendorDir/" . $packageName;
+
+            if (file_exists("$packagePath/public")) {
+                $this->copyDirectory("$packagePath/public", "$projectDir/public", $io);
+            }
+
+            if (file_exists("$packagePath/skin")) {
+                $this->copyDirectory("$packagePath/skin", "$projectDir/public/skin", $io);
+            }
+
+            if (file_exists("$packagePath/js")) {
+                $this->copyDirectory("$packagePath/js", "$projectDir/public/js", $io);
+            }
```

Comment:
How about `$packagePath/media` as well?

Reply:
Although I know that sort of contradicts my earlier statement on overwriting files. But I do know modules put files here....

Reply:
mmmm I kinda lean towards not copying /media because it's true that's not usually under versioning so it could be problematic in production. but also, could it be considered a bad practice to wanting to add files to /media?

Reply:
Yeah, I think we can skip it.

Reply:
:-)